### ISSUE TITLE
Hide the filling up of the scoreboard?

### ISSUE THREAD

**Issue by Cosmiiko** [MrMicky-FR/FastBoard#10]
When showing the Scoreboard to the player (using new Fastboard(player)), the user can see the board filling up as illustrated here:
![GIF](https://i.gyazo.com/7c705bad61cf4660439c06a1ba6faa3a.gif)

Is there any way to circumvent that? Like filling up the board with data, and only then show it to the player?

**Comment by MrMicky-FR** [comment]
Hi, can you send the code you use to create the scoreboard ? Because I can't reproduce this issue

**Comment by Cosmiiko** [comment]
Ah, just realised why you can't reproduce it.
It has to do with ViaVersion. My client is on 1.16 while the server is running 1.8.8. I guess it must mess with the packets somehow...
When I switched my client to 1.8.8, the scoreboard appears at once, as expected. Should've tried before opening this issue.

I'm guessing you have no control at all over this, thanks for answering so quickly anyway!

(That being said if you think this is fixable somehow, I'm all ears, code used below:

In a command that the player can run:
```Java
scoreboards.put(pl.getUniqueId(), new FastBoard(pl));
for (var board : scoreboards.values()) {
    board.updateTitle("§c§lBedwars");
    board.updateLines(Arrays.asList(
            "This is",
            "a very",
            "long board",
            "just to",
            "show how",
            "you can",
            "see it",
            "fill itself",
            "up."
    ));
}
```
)

**Comment by MrMicky-FR** [comment]
I tested with ViaVersion and I still can’t reproduce the issue.
The code looks good, maybe try to see if your server/proxy is not overloaded because lags could cause something like this

**Comment by Cosmiiko** [comment]
Okay I am so confused right now.
This was caused by a bug with Lazy Chunk Loadings in Optifine HD U G7.

Updated to G8, scoreboard appears perfectly. Switched back to G7, bug occurs.
Took a peek at Optifine's changelog for G8:
> - fixed input lag with lazy chunk loading (#5348)

The bug went away after I disabled Lazy CL in G7.

Guess Optifine's lag affected UI too.

(Tried with plenty of other MC versions / configurations, this was indeed the issue)
